app/views/schools/school_targets/progress/_charts.html.erb

```diff
@@ -13,14 +13,14 @@
 </ul>
 <div class="tab-content" id="targets-overview-tabs">
   <div class="tab-pane fade show active" id="cumulative-progress-<%= fuel_type.to_s %>" role="tabpanel" aria-labelledby="cumulative-progress-tab">
-    <%= render 'schools/analysis/chart_name', school: @school, content: "targeting_and_tracking_weekly_#{fuel_type.to_s}_to_date_cumulative_line".to_sym, mpan_mprn: nil %>
```

Comment:
We were still referring to an older partial rather than using the chart component so have switched to that as part of removing the partial